cmd/query/app/querysvc/adjuster/sort.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package adjuster
+
+import (
+	"sort"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+var _ Adjuster = (*SortAttributesAndEventsAdjuster)(nil)
+
+// SortAttributesAndEvents creates an adjuster that standardizes trace data by sorting elements:
+// - Resource attributes are sorted lexicographically by their keys.
+// - Span attributes are sorted lexicographically by their keys.
+// - Span events are sorted lexicographically by their names.
+// - Attributes within each span event are sorted lexicographically by their keys.
+func SortAttributesAndEvents() SortAttributesAndEventsAdjuster {
+	return SortAttributesAndEventsAdjuster{}
+}
+
+type SortAttributesAndEventsAdjuster struct{}
+
+func (s SortAttributesAndEventsAdjuster) Adjust(traces ptrace.Traces) {
+	resourceSpans := traces.ResourceSpans()
+	for i := 0; i < resourceSpans.Len(); i++ {
+		rs := resourceSpans.At(i)
+		resource := rs.Resource()
+		s.sortAttributes(resource.Attributes())
+		scopeSpans := rs.ScopeSpans()
+		for j := 0; j < scopeSpans.Len(); j++ {
+			ss := scopeSpans.At(j)
```

Comment:
why not sort scope attributes?
```suggestion
			ss := scopeSpans.At(j)
			s.sortAttributes(ss.Attributes())
```

Reply:
@yurishkuro this is something I wanted to ask. Do we want to sort all possible attributes? or just the ones that are used in model to otel translation?